### PR TITLE
Bobcat dev

### PR DESCRIPTION
**Pull request by robbrockbank**

### CODE REVIEW

File: cookbooks/calico/attributes/default.rb [hunk 1]
```diff
@@ -7,3 +7,11 @@
 default["calico"]["package_source"] = "http://binaries.projectcalico.org/repo ./"
```
Comment (lwr20) [inline cookbooks/calico/attributes/default.rb]: Is this still the correct default?

Comment (lwr20) [inline cookbooks/calico/attributes/default.rb:7]: Is this still the right default?

File: cookbooks/calico/recipes/control.rb [hunk 2]
```diff
@@ -313,7 +343,7 @@
     user "root"
     environment node["run_env"]
     code <<-EOH
-    wget http://cdn.download.cirros-cloud.net/0.3.2/cirros-0.3.2-x86_64-disk.img -O - | glance image-create --name=cirros-0.3.2-x86_64 --disk-format=qcow2 \
+    wget http://calico-jenkins.datcon.co.uk/images/cirros-0.3.2-x86_64-disk.img -O - | glance image-create --name=cirros-0.3.2-x86_64 --disk-format=qcow2 \
```
Comment (lwr20) [inline cookbooks/calico/recipes/control.rb:346]: Change this back - this is internal only!

**Comment by tomdee** [comment]
Closing v. old PR.

**Pull request closed without merging**